guides/orders/overview.md

```diff
@@ -0,0 +1,162 @@
+# Overview
+
+The `Spree::Order` model is one of the key models in Solidus. It provides a
+central place around which to collect information about a customer order. It
+collects line items, adjustments, payments, addresses, return authorizations,
+and shipments. 
+
+Whenever information about the order is updated, the `Spree::OrderUpdater`
+should update the order. If your store creates or changes the functionality
+around the order, you may need to [manually call the order
+updater][update-orders].
+
+Orders have the following attributes:
+
+- `number`: The unique identifier for this order. It begins with the letter `R`
+  and ends in a nine-digit number (for example, `R123456789`). This number is
+  shown to the users, and can be used to find the order by calling
```

Comment:
maybe here we should link the [`Order::NumberGeneratore`](https://github.com/solidusio/solidus/blob/c74c7d0351cf678406dee218a19070756bcbfbbf/core/app/models/spree/order/number_generator.rb#L19) which allows to change how this number is generated. WDYT?

Reply:
Done! Thanks for the suggestion.